scm_v3c/radio.c

```diff
@@ -18,9 +18,11 @@ unsigned int acfg3_val;
 
 // These coefficients are used for filtering frequency feedback information
 // These are no necessarily the ideal values to use; situationally dependent
-unsigned char FIR_coeff[11] = {4,16,37,64,87,96,87,64,37,16,4};
-unsigned int IF_estimate_history[11] = {500,500,500,500,500,500,500,500,500,500};
-signed short cdr_tau_history[11] = {0};
+#define FILTER_WINDOWS_LEN       11
+
+uint8_t FIR_coeff[FILTER_WINDOWS_LEN] = {4,16,37,64,87,96,87,64,37,16,4};
```

Comment:
How the coeff arrary is determined?

Reply:
Hmm, I used the same filter that Brad used to have for his `radio_housekeeping()` function.  I can email him.